lib/src/test/java/com/knuddels/jtokkit/reference/Cl100kBaseTest.java

```diff
@@ -14,22 +14,26 @@ class Cl100kBaseTest {
 
     private static final Encoding ENCODING = Encodings.newDefaultEncodingRegistry().getEncoding(EncodingType.CL100K_BASE);
 
+    Encoding getEncoding() {
+        return ENCODING;
```

Comment:
we have to be able to override it from a child test to control the order of initializations